docs/xrefs.json

```diff
@@ -707,6 +707,10 @@
     "title": "RX0000",
     "path": "/rsh/errors/#RX0000"
   },
+  "build": {
+    "title": "Reach is for Builders",
+    "path": "/build/#build"
```

Comment:
This pull request needs the `build/` directory.

Reply:
`build/` has been restored.